src/Sarif/Core/RunAutomationDetails.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    /// <summary>
+    /// Information that describes a run's identity and role within an engineering system process.
+    /// </summary>
+    public partial class RunAutomationDetails
+    {
+        public bool ShouldSerializeDescription() => this.Description != null;
```

Comment:
>Description [](http://example.com/codeflow?start=57&length=11)

?? why persist an empty description? how is this useful?

Reply:
The Description isn't a string, it is a Message type. It's why I didn't change to NullOrWhiteSpace check